flink-runtime/src/main/java/org/apache/flink/runtime/io/network/netty/NettyMessageDecoder.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.runtime.io.network.netty;
+
+import org.apache.flink.shaded.netty4.io.netty.buffer.ByteBuf;
+import org.apache.flink.shaded.netty4.io.netty.channel.ChannelHandlerContext;
+
+/**
+ * Parsers for specified netty messages.
+ */
+abstract class NettyMessageDecoder implements AutoCloseable {
+
+	/** ID of message currently under decoding. */
+	protected int msgId;
+
+	/** Length of message currently under decoding */
+	protected int messageLength;
+
+	/**
+	 * The result of message parsing with the provided data.
+	 */
+	static class ParseResult {
```

Comment:
ParseResult -> DecodingResult ?

Reply:
Agree with changing the name and have fixed.